tests/testthat/test-parameter-identification.R

```diff
@@ -71,11 +71,11 @@ test_that("ParameterIdentification configuration can be changed without error",
 test_that("ParameterIdentification$run() errors on invalid objective function option", {
   piTask <- createPiTask()
   piTask$configuration$objectiveFunctionOptions$objectiveFunctionType <- "invalidType"
-  expect_error(piTask$run(), "Invalid value for objectiveFunctionType")
+  expect_error(piTask$run())
```

Comment:
Why do you want to stop checking for a specific error?

Reply:
Because making sure the right message is sent back is ospsuite.utils job so it must be tested there.
Here, we just want to make sure that an unsuported option return an error.